duckdb-vortex/src/vortex_extension.cpp

```diff
@@ -265,11 +268,17 @@ static void VortexScanFunction(ClientContext &context, TableFunctionInput &data,
 		local_state.current_row = 0;
 	}
 
+	if (local_state.cache == nullptr) {
+		// Create a unique value so each cache can be differentiated.
+		local_state.cache = ConversionCache_create(++global_state.unique_id);
```

Comment:
Could consider post-increment if you want the first id to be `0` => `global_state.unique_id++`